### PR TITLE
Updating the Gubernator documentation to correctly link to the Login page for ACK.

### PR DESCRIPTION
**Pull request by mithrav**

### CODE REVIEW

File: gubernator/templates/pr_dashboard.html [hunk 1]
```diff
@@ -63,7 +63,7 @@ <h3 class="pr-section">
 % endfor
 
 <div id="help">
-<p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. Use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
+  <p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. <a href="/pr">Login</a> to use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: Make the login prompt only show if the user isn't logged in:

    {% if user != login or not login %}<a href="/pr">Login</a> to use{% else %}Use{% endif %}

**Comment by BenTheElder** [comment]
@mithrav your commits don't seem to be showing up under your `@google.com` email, otherwise they'd be under the CLA already and each commit would show your avatar.

**Comment by rmmh** [comment]
`git config --global user.email mithra@google.com`
`git filter-branch --env-filter 'export GIT_COMMITTER_EMAIL=mithra@google.com GIT_AUTHOR_EMAIL=mithra@google.com' master..HEAD' 

`From: Mithra Seshadrirajah <mithra@mithrapony.mtv.corp.google.com>`

**Comment by mithrav** [comment]
PTAL now

File: gubernator/templates/pr_dashboard.html [hunk 2]
```diff
@@ -63,7 +63,8 @@ <h3 class="pr-section">
 % endfor
 
 <div id="help">
-<p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. Use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
+  <p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>.
+   {% if user != login or not login %}<a href="/pr">Login</a> to use {% else %}Use {% endif %}<a href="/pr">Login</a> the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: Remove the second "Login"

Comment (mithrav) [reply gubernator/templates/pr_dashboard.html]: Done

File: gubernator/templates/pr_dashboard.html [hunk 3]
```diff
@@ -63,7 +63,8 @@ <h3 class="pr-section">
 % endfor
 
 <div id="help">
-<p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://git.k8s.io/test-infra/gubernator/github/classifier.py#L397">simple state machine</a>. Use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
-<p>Have a problem? <a href="https://github.com/kubernetes/test-infra/issues/new?title=PR%20Dashboard:&labels=area/gubernator&assignee=rmmh">File a bug!</a>
+  <p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>.
+  {% if user != login or not login %}<a href="/pr">Login</a> to use {% else %}Use {% endif %} the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
+<p>Have a problem? <a href="https://github.com/kubernetes/test-infra/issues/new?title=PR%20Dashboard:&labels=gubernator&assignee=rmmh">File a bug!</a>
```
Comment (rmmh) [inline gubernator/templates/pr_dashboard.html]: retain `labels=area/gubernator`

**Comment by rmmh** [comment]
/hold
/lgtm

please squash before /hold cancel

**Comment by stevekuznetsov** [comment]
@mithrav you'll want to `git rebase` on top of latest `master` so this patchset doesn't contain other diffs

**Pull request merged**